sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -294,17 +294,16 @@ private[sql] abstract class BaseWriterContainer(
 
   private def newOutputCommitter(context: TaskAttemptContext): OutputCommitter = {
     val committerClass = context.getConfiguration.getClass(
-      "mapred.output.committer.class", null, classOf[OutputCommitter])
+      "mapreduce.output.committer.class", null, classOf[OutputCommitter])
```

Comment:
Because we are using APIs in mapreduce package, we cannot use `mapred.output.committer.class`. I am just creating another conf flag called `mapreduce.output.committer.class` (it is not defined in hadoop).